lib/services/database/firestore_service.dart

```diff
@@ -21,4 +23,23 @@ class FirestoreService implements DatabaseService {
         .doc('players/$userId')
         .set(<String, String>{'username': username}, SetOptions(merge: true));
   }
+
+  @override
+  Stream<List<Game>> get streamOfCreatedGames =>
+      _firestore.collection('games').snapshots().map((QuerySnapshot snapshot) {
```

Comment:
some sweet functional programming here!  I don't think there's any kind of rule about it but functional programming reads better with fat arrow syntax I reckon... maybe check it out and see if you agree :-)